source/features/add-updated-date-to-pinned-issues.tsx

```diff
@@ -0,0 +1,69 @@
+import clockIcon from 'octicon/clock.svg';
+import features from '../libs/features';
+import * as api from '../libs/api';
+import {getRepoGQL} from '../libs/utils';
+import React from 'dom-chef';
+import select from 'select-dom';
+
+async function init(): Promise<false | void | any> {
+	// Get issues links that don't already have a specific sorting applied
+	for (const link of select.all<HTMLAnchorElement | any>('.pinned-issue-item')) {
+		const issueNUmber: any = Number((select<any>('a.link-gray-dark.no-underline.h4[href]', link)).href.split('/').slice(-1).pop());
```

Comment:
This is a shortest way to get the number; and it can stay a string
```suggestion
		const issueNumber = select
$('.opened-by', issueNumber)!.firstChild!.textContent!.replace(/\D/g , '');
```